client-app/src/pages/account/views/AccountOrderDetailsModal.vue

```diff
@@ -0,0 +1,143 @@
+          <b-card-body>
+            <div class="row">
+              <div class="col-md-6">
+                <div class="list-group list-group-flush">
+                  <div class="list-group-item d-flex justify-content-between">
+                    <span class="font-weight-bold">{{ $t("account.orders.order-details.subtotal") }}</span>
+                    {{ subtotal }}
+                  </div>
+                  <div class="list-group-item d-flex justify-content-between">
+                    <span class="font-weight-bold">{{ $t("account.orders.order-details.shipping") }}</span>
+                    {{ shipping }}
+                  </div>
+                  <div class="list-group-item d-flex justify-content-between">
+                    <span class="font-weight-bold">{{ $t("account.orders.order-details.total") }}</span>
+                    {{ total }}
+                  </div>
+                </div>
+              </div>
+              <div class="col-md-6">
+                <div class="list-group list-group-flush">
+                  <div class="list-group-item d-flex justify-content-between">
+                    <span class="font-weight-bold">{{ $t("account.orders.order-details.created-by") }}</span>
+                    {{ createdBy }}
+                  </div>
+                  <div class="list-group-item d-flex justify-content-between">
+                    <span class="font-weight-bold">{{ $t("account.orders.order-details.status") }}</span>
+                    {{ status }}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </b-card-body>
+        </b-collapse>
+      </b-card>
+
+      <b-card no-body class="mb-1">
+        <b-card-header header-tag="header"
+                       class="p-1"
+                       role="tab">
+          <b-button
+            v-b-toggle.accordion-2
+            block
+            variant="primary">
+            {{ $t("account.orders.order-details.order-details") }}
+          </b-button>
+        </b-card-header>
+        <b-collapse id="accordion-2"
+                    accordion="my-accordion"
+                    role="tabpanel">
+          <b-card-body v-for="item in order.items" :key="item.id">
+            <b-card-text>{{ item.name }}</b-card-text>
+            <b-card-text>
+              <span class="font-weight-bold">{{ $t("account.orders.order-details.sku") }}</span>
+              {{ item.sku }}
+            </b-card-text>
+            <b-card-text>
+              <span class="font-weight-bold">{{ $t("account.orders.order-details.unit-price") }}</span>
+              {{ item.placedPrice.formattedAmount }}
+            </b-card-text>
+            <b-card-text>
+              <span class="font-weight-bold">{{ $t("account.orders.order-details.quantity") }}</span>
+              {{ item.quantity }}
+            </b-card-text>
+          </b-card-body>
+        </b-collapse>
+      </b-card>
+    </div>
```

Comment:
I guess we should extract the dummy component for OrderDetails that receive the order of type CustomerOrder on input as property.